rspace/src/main/scala/coop/rchain/rspace/nextgenrspace/history/HistoryRepositoryImpl.scala

```diff
@@ -192,6 +234,15 @@ object HistoryRepositoryImpl {
       .get
       .toByteVector
 
+  private def encodeUnsorted[D](data: Seq[D])(implicit codec: Codec[D]): ByteVector =
```

Comment:
this looks out of place, shouldn't it be part of dev?

Reply:
yes. that is part of dev branch. strange!

Reply:
`git rebase`?